.github/workflows/ci.yml

```diff
@@ -24,30 +24,31 @@ jobs:
     name: test
     env:
       RUST_VERSION: ${{ matrix.rust }}
+      TARGET: ${{ matrix.target }}
     strategy:
       matrix:
-        crates:
-          - crossbeam
-          - crossbeam-channel
-          - crossbeam-deque
-          - crossbeam-epoch
-          - crossbeam-queue
-          - crossbeam-skiplist
-          - crossbeam-utils
-        rust:
-          - 1.36.0
-          - nightly
-        os:
-         - ubuntu-latest
-         - windows-latest
-    runs-on: ${{ matrix.os }}
+        include:
```

Comment:
It seems we don't test Crossbeam for Rust stable.  Is it intended?

Reply:
I don't think it was intended. Added the latest stable Rust to matrix.